Automattic/wp-calypso
# Help Center: Show help center in checkout

#### Proposed Changes

We are releasing the help center in `wordpress.com/checkout/YOURSITE.wordpress.com`.
Also:
- Changed the Help Center loading to be async, so separating it from the checkout chunk.
- Moved the logic for deciding to show the Help Center or not


#### Testing Instructions



* Checkout branch
* `yarn start`
* Log with a user with id < 10, to not see the FAB but only the Help Center.
* Visit `http://calypso.localhost:3000/checkout/YOURSITE.wordpress.com` and check that that you see the help center and not the FAB
